services/scabbard/cli/src/main.rs

```diff
@@ -778,8 +778,20 @@ fn run() -> Result<(), CliError> {
                     .with_contract(smart_contract.contract)
                     .into_payload_builder()?
                     .into_transaction_builder(&*signer)?
-                    .into_batch_builder(&*signer)?
-                    .build(&*signer)?;
```

Comment:
Doesn't this mean there is already a From implementation?

Reply:
Yes but that is for the 0.2 version of transact error which is used above in SmartContractArchive. This is the 0.3 transact error because that's the version sabre sdk is using.

Reply:
Oh, I see.  Makes sense now.